dartagnan/src/main/java/com/dat3m/dartagnan/verification/model/event/DefaultEventModel.java

```diff
@@ -0,0 +1,85 @@
+package com.dat3m.dartagnan.verification.model.event;
+
+import com.dat3m.dartagnan.program.event.core.*;
+import com.dat3m.dartagnan.program.event.*;
+import com.dat3m.dartagnan.verification.model.ThreadModel;
+
+import static com.dat3m.dartagnan.program.event.Tag.*;
+
+
+public class DefaultEventModel implements EventModel {
+    protected final Event event;
+    protected final ThreadModel thread;
+    protected int id = -1;
```

Comment:
The id is always initialized by the constructor. This "default" value is useless.